pages/githubissues.md

```diff
@@ -8,14 +8,18 @@ First, you will need to open an issue within the right repository ([upstream rep
 
 After this, you will then research the possible solutions or ideas on how to address the issue and write a comment with this information, as you can see [here](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/15). Someone else will hopefully provide input (as Dogi did here) and then you can move ahead with making the proposed changes.
 
-Every time you start to work on an issue you need to create a branch to keep your work on various issues separate from each other. Before you create your branch, sync your repo following the process in the [previous step](http://open-learning-exchange.github.io/#!pages/gitandrepositories.md). Then use  `git checkout -b name_of_your_new_branch --track origin/name_of_your_new_branch` to create and switch to a new branch.
+Every time you start to work on an issue you need to create a branch to keep your work on various issues separate from each other. Before you create your branch, sync your repo following the process in the [previous step](http://open-learning-exchange.github.io/#!pages/gitandrepositories.md). Then use  `git checkout -b name_of_your_new_branch` to create and switch to a new branch on your local repository. Then create an initial commit `git commit -m "commit message"` and finally push the newly created local branch using `git push -u origin new-branch` to your remote repository.
+
+**TIP**: The default branch in your repository is called master. Keeping the master branch clean of any changes is a good practice so if you make any changes you dont want, you may revert back.
 
```

Comment:
I would organize these changes a bit differently.

After:
Every time you start to work on an issue you need to create a branch to keep your work on various issues separate from each other. Before you create your branch, sync your repo following the process in the [previous step](http://open-learning-exchange.github.io/#!pages/gitandrepositories.md).

I would say that there are two ways of doing this,
1. create a branch in your remote repository by following the instructions [here](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/), then use 'git checkout -b name_of_your_new_branch --track origin/name_of_your_new_branch' to switch to that branch on your local repository.
2. use  `git checkout -b name_of_your_new_branch` to create and switch to a new branch on your local repository, then following the instructions below to create a new commit. and finally use `git push -u origin name_of_your_new_branch` to push it to your remote repository and track it.